Userland/Libraries/LibC/sys/socket.cpp

```diff
@@ -112,6 +112,12 @@ ssize_t recvfrom(int sockfd, void* buffer, size_t buffer_length, int flags, stru
     return rc;
 }
 
+int getnameinfo(const struct sockaddr*, socklen_t, char*, socklen_t, char*, socklen_t, int)
+{
+    // FIXME: Actually implement this
```

Comment:
```suggestion
    VERIFY_NOT_REACHED();
```
More like this?

Reply:
if anything, more like this:

```suggestion
    TODO();
```